vimeo/exceptions.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+
+class BaseVimeoException(Exception):
+
+    def __init__(self, request, message):
+
+        json = None
+        try:
+            json = request.json() 
+        except:
+            pass
```

Comment:
If we can't parse the json we should use the request body as the entire error message (proxy problems etc. could cause this)